regression/cbmc/Bitfields3/main.c

```diff
@@ -0,0 +1,24 @@
+#include <stdlib.h>
+#include <assert.h>
+
+#pragma pack(1)
+struct A {
+  unsigned char a;
+  unsigned char b:2;
```

Comment:
You should probably also test the case where bitfields and "normal" fields are interleaved, e.g. `<byte, 2bit, byte, 2bit>`. Also the edge case where the bitfield precisely equals a byte (i.e. `<nybble, nybble>` should have size 1)

Reply:
Done.